pom.xml

```diff
@@ -10,6 +10,7 @@
     <module>${project.name}</module>
     <module>${project.name}-demo</module>
     <module>${project.name}-test</module>
+    <module>product</module>
```

Comment:
${project.name}-product  looks much better IMHO ... I'd always prefer a meaningful name in the directories

Reply:
still we could opt for a 'product' directory in the template ... doesn't hurt us I think

Reply:
renamed to excel-connector-product